core/settings/general/manager.php

```diff
@@ -33,7 +33,7 @@ class Manager extends CSS_Manager {
 	public function __construct() {
 		parent::__construct();
 
-		// TODO: _deprecated_file( __FILE__, '3.0.0', 'Plugin::$instance->kits_manager->get_active_kit_for_frontend()' );
+		_deprecated_file( __FILE__, '3.0.0', 'Plugin::$instance->kits_manager->get_active_kit_for_frontend()' );
```

Comment:
@davseve can you replace this with
`Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation.....`

Reply:
I have seen mentions to `_deprecated_file` method,
can you please write exactly what you want me to replace it with?
do you mean `Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_file( __FILE__, '3.0.0', 'Plugin::$instance->kits_manager->get_active_kit_for_frontend()' );`